test/use-package-test.el

```diff
@@ -1,4 +1,4 @@
-;;; use-package-tests.el --- Tests for use-package.el
+;;; use-package-test.el --- Tests for use-package.el
```

Comment:
The test files that come with Emacs all use `-tests.el`(e.g., http://git.savannah.gnu.org/cgit/emacs.git/tree/test/lisp). It's too bad that the tools outside of Emacs decided on a different convention.